nix-darwin/default.nix

```diff
@@ -28,4 +28,3 @@ in {
         '') cfg.users);
     })
   ];
-}
```

Comment:
I assume this is a mistake?

Reply:
Yes. Failed to check the diff after rebasing : /